app/views/users/phone_setup/spam_protection.html.erb

```diff
@@ -23,8 +23,13 @@
       class="g-recaptcha"
       data-sitekey="<%= IdentityConfig.store.recaptcha_site_key_v2 %>"
       data-callback="onCaptchaResponse"
+      data-action="<%= PhoneRecaptchaValidator::RECAPTCHA_ACTION %>"
     ></div>
-    <script src="https://www.google.com/recaptcha/api.js" async></script>
+    <% if FeatureManagement.recaptcha_enterprise? %>
+      <script src="https://www.google.com/recaptcha/enterprise.js" async></script>
+    <% else %>
+      <script src="https://www.google.com/recaptcha/api.js" async></script>
+    <% end %>
```

Comment:
should we load neither of these if the mock is enabled?

Reply:
Since this is inside the condition for a configured site key being present, it might have been assumed that you'd _either_ configure credentials _or_ use the mock validator, but never both.